docs/docs/validating-plugin-options.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Validating plugin options
+---
+
+To help users [configure plugins](/docs/configuring-usage-with-plugin-options/) correctly, a plugin can optionally define a schema to enforce a type for each option. Gatsby will validate that the options users pass match the schema to help them correctly set up their site.
+
+## How to define an options schema
+
+To define their options schema, plugins export [`pluginOptionsSchema`](/docs/node-apis/#pluginOptionsSchema) from their `gatsby-node.js` file. This function gets passed an instance of [Joi](https://joi.dev) to define the schema with.
```

Comment:
Could this get converted into a numbered list of steps? This feels like the core idea behind this "how to" guide, and right now it's a little sparse. Someone might skip over it without realizing that those are the steps they're meant to be following.

Reply:
I tried that but it felt very strange, as it's really only one step 😅 At least I made this paragraph a tiny bit meatier...